src/components/Square.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
+  // For Wave 1 enable this
+  //  Component to alert a parent
   //  component when it's clicked on.
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  return (
+    <button
+      className="square"
+      onClick={() => {
+        props.onClickCallback({ id: props.id });
+      }}
+    >
+      {props.value}
+    </button>
+  );
+};
```

Comment:
Great work using an anonymous callback function for `onClick`. 

We could also use object destructing on `props` to use the key names directly:


```
const Square = ( {value, id, onClickCallback, key} ) => {
 return (
    <button
      className="square"
      onClick={() => {
        onClickCallback({ id: id });
      }}
    >
      {value}
    </button>
  );
};

```